config/crds/hive_v1_clusterdeployment.yaml

```diff
@@ -226,7 +226,7 @@ spec:
                         will be created.
                       type: string
                   type: object
-                baremetal:
+                bareMetal:
```

Comment:
This looks to be undoing an intentional change, this should be "baremetal"

Reply:
Yep. Must have snuck in with the rebase.